libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -86,7 +89,9 @@ function parseCommandOutput(payload: InvokeCommandOutput): any | undefined {
       throw newError;
```

Comment:
Should we include the log link for this case as well? (where we extract the error message properly)

Reply:
Good call